### ISSUE TITLE
Performance of "standalone_model_numpy.py" vs "standalone_model_tf.py"

### ISSUE THREAD

**Issue by jannisborn** [connorcoley/scscore#10]
From your code, I see that the numpy model is a leightweight standalone version of the tensorflow version. I would like to understand how much these two models differ in terms of accuracy. In your paper you mention that tensorflow and python2 were used to train the model, so I reckon the "standalone_model_tf.py" is identical to the model used to produce the results in the paper. I want to integrate your model into my code, but I'm unsure about which option to choose. If leightweight  comes at high costs of performance decline, I would rather take the tensorflow model.

Maybe you can give a rough estimate or an intuition how well you expect the numpy model as opposed to the tensorflow one. 

Thanks!

**Comment by jannisborn** [comment]
Ah, never mind, just saw this is the same checkpoint, you just converted the weights from tf to numpy.

**Comment by jannisborn** [comment]
For your reference, I just integrated your code into our [PaccMann^RL project](https://arxiv.org/abs/1909.05114):  https://github.com/PaccMann/paccmann_generator/commit/6a183c55ca70e4f0cbf93a9f45aa82868364925b
